spec/models/beta/search/intercept_message_spec.rb

```diff
@@ -76,5 +76,18 @@
 
       it { is_expected.to eq(expected_message) }
     end
+
+    context 'when there is a section followed by a roman number' do
+      subject(:formatted_message) do
+        build(:intercept_message,
```

Comment:
We should chuck this in a factory to be consistent

Reply:
tbh I was considering the usefulness of the factories in this case,
I was always jumping back and forth in the factory/test, and it make the test couple with the factory 🤔

Reply:
I'm happy to change to not using traits but I'd make the rest conform to this new standard rather than have a mixture, etc